### PR TITLE
[RFC][UN-7148] Tabs if the user goes back - ember components

### PR DESCRIPTION
**Pull request by mmelo**
Refers to: https://uniplaces.atlassian.net/browse/UN-7148

Created uni-tabs component

### CODE REVIEW

File: addon/templates/components/uni-tabs.hbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="uni-tabs__wrapper">
+    {{#each tabs as |item|}}
+        <div class="uni-tabs__wrapper__item">
+            <a href="{{item.url}}">{{item.label}}</a>
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-tabs.hbs]: this should be a link-to. this way, when a specific route is active, it automatically triggers the active class.
https://guides.emberjs.com/v2.8.0/templates/links/

File: addon/templates/components/uni-tabs.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="uni-tabs__wrapper">
+    {{#each tabs as |item|}}
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-tabs.hbs]: each tabs as tab seems more correct

File: tests/integration/components/uni-tabs-test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('uni-tabs', 'Integration | Component | uni tabs', {
```
Comment (AndreJoaquim) [inline tests/integration/components/uni-tabs-test.js:6]: Please make some tests :)

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Comment by mmelo** [comment]
@raffo0707 @montoias @AndreJoaquim fixes made

**Comment by AndreJoaquim** [comment]
Very good job! 💯 / 💯

**Review by AndreJoaquim — APPROVED** [review]

**Review by montoias — APPROVED** [review]

**Pull request merged**